src/components/TodoContainer.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import TodoList from "./TodosList";
+import Header from "./Header";
+import InputTodo from "./InputTodo";
+import { v4 as uuidv4 } from "uuid";
+
+class TodoContainer extends React.Component {
+  state = {
+    todos: [
+      {
+        id: uuidv4(),
+        title: "Setup development environment",
+        completed: true,
+      },
+      {
+        id: uuidv4(),
+        title: "Develop website and add content",
+        completed: false,
+      },
+      {
+        id: uuidv4(),
+        title: "Deploy to live server",
+        completed: false,
+      },
+    ],
+  };
+
+  handleChange = (id) => {
+    this.setState((prevState) => ({
+      todos: prevState.todos.map((todo) => {
+        if (todo.id === id) {
+          return {
+            ...todo,
+            completed: !todo.completed,
+          };
+        }
+        return todo;
+      }),
+    }));
+  };
+
+  delTodo = (id) => {
+    this.setState({
+      todos: [
+        ...this.state.todos.filter((todo) => {
+          return todo.id !== id;
+        }),
+      ],
+    });
+  };
+
+  addTodoItem = (title) => {
+    const newTodo = {
+      id: 4,
+      title: title,
+      completed: false,
+    };
+    this.setState({
+      todos: [...this.state.todos, newTodo],
+    });
+  };
```

Comment:
- this is your problem, id: 4, which means any time you create a new task the id will be 4.
- so you can use the `uuidv4()` 
the code will be like this: 
```
const newTodo = {
      id: uuidv4(),
      title: title,
      completed: false,
    };
```
**now all new tasks have deference id.**